app/src/main/java/de/triology/cas/logging/DefaultMappingPasswordRewritePolicy.java

```diff
@@ -33,7 +33,7 @@ protected String getPasswordFlag() {
     protected String replacePasswordValue(String originMessage) {
         String modifiedMessage = null;
         if (originMessage != null) {
-            modifiedMessage = originMessage.replaceAll("mappedValue = '\\S+'", "mappedValue = '****'");
+            modifiedMessage = originMessage.replaceAll("mappedValue = array<Character>\\[.*\\]", "mappedValue = '****'");
```

Comment:
Why can we not just use a generic solution like this?  
```suggestion
            modifiedMessage = originMessage.replaceAll("mappedValue = .*", "mappedValue = '****'");
```

Reply:
With this change you will delete all chars after the mappedValue.

Reply:
yeah, I just noticed that, too... But there has to be a solution to this right? I mean it's regex :D